src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -287,7 +298,7 @@ def send_message(
     return {
         "id": response["channel"],
         "timestamp": response["ts"],
-        "weblink": f"https://slack.com/app_redirect?channel={response['id']}",  # TODO should we fetch the permalink?
+        "weblink": get_message_permalink(client, response["channel"], response["ts"]),
```

Comment:
it'd be good to cache `get_message_permalink` w/  the channed id if we'll call it on each `send_message`

Reply:
@wssheldon I may be missing something, but why would we want to cache here, if each message will have a different permalink?

We could also not do the API call and include the permalink in the response, because we're not doing anything with it at the moment.

Reply:
Ah yes, you're right. I'm surprised `chat_post_message` doesn't just return the permalink in the response. We should be able cache the domain we fetch on conversation creation though right?

`domain = make_call(client, SlackAPIGetEndpoints.team_info)["team"]["domain"]`

Reply:
Yeah, that one we could cache.

Reply:
Done.